lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -236,6 +242,68 @@ void PECOFFImporter::parseImportedSymbols() {
   }
 }
 
+void PECOFFImporter::recordDelayImportedFunction(
+  const llvm::object::DelayImportDirectoryEntryRef &I,
+  iterator_range<llvm::object::imported_symbol_iterator> Range) {
```

Comment:
If `iterator_range` is making your life hard with `revng-check-conventions` consider stuffing an `auto` instead of the full type (or just go for a `using`).